store/src/java/com/zimbra/cs/volume/VolumeCLI.java

```diff
@@ -123,6 +152,11 @@ private void setArgs(CommandLine cl) throws ServiceException, ParseException, IO
         volumePrefix = cl.getOptionValue(O_VP);
         storageType = cl.getOptionValue(O_STP);
         bucketId = cl.getOptionValue(O_BID);
+        url = cl.getOptionValue(O_URL);
```

Comment:
You should add conditions here for OpenIO.

Reply:
Okay, Done.!